lms/djangoapps/courseware/tests/helpers.py

```diff
@@ -367,17 +367,17 @@ def get_expiration_banner_text(user, course, language='en-us'):
     if upgrade_deadline is None or now() < upgrade_deadline:
         upgrade_deadline = enrollment.course_upgrade_deadline
 
-    language_is_es = language and language.split('-')[0].lower() == 'es'
-    if language_is_es:
-        formatted_expiration_date = strftime_localized(expiration_date, '%-d de %b. de %Y').lower()
-    else:
-        formatted_expiration_date = strftime_localized(expiration_date, '%b. %-d, %Y')
-
+    date_string = '<span class="localized-datetime" data-format="shortDate" \
+        data-datetime="{formatted_date}" data-language="{language}">{formatted_date}</span>'
+    formatted_expiration_date = date_string.format(
+        language=language,
+        formatted_date=strftime_localized(expiration_date, '%b. %-d, %Y')
```

Comment:
I thought the JS approach was to render the date in english, and then do all of the translation client side?

Reply:
More concretely, wondering if we should be calling just `strftime` here instead of `strftime_localized`

Reply:
strftime would work although if we use strftime_localized we'll still have a translated string if the js fails for some reason. perhaps we want the backup string to be localized?  
don't have a strong opinion either way though

Reply:
Interesting, so the JS doesn't require the input to be in english?

Reply:
the JS just uses the data-attributes and replaces whatever is in the span (i will double check)